alex/task_1/task1/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,26 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { HomeComponent } from './home/home.component';
+import { NotFoundComponent } from './not-found/not-found.component';
+
+const routes: Routes = [
+  {
+    path: '',
+    component: HomeComponent
+  },
+  {
+    path: 'dashboard',
+    loadChildren: () => import('./dashboard/dashboard-routing.module').then(m => m.DashboardRoutingModule)
+  },
+  {
+    path: 'profile',
+    loadChildren: () => import('./profile/profile-routing.module').then(m => m.ProfileRoutingModule)
```

Comment:
as per Angular documentation

https://angular.io/guide/lazy-loading-ngmodules#lazy-loading-basics
`import('./profile/profile.module').then(m => m.ProfileModule)`

should be used instead.

Reply:
Same is applied for all lazy routes. Please refactor

Reply:
Done